alchemist-cognitive-agents/src/main/kotlin/it/unibo/alchemist/model/implementations/nodes/AbstractHomogeneousPedestrian.kt

```diff
@@ -30,6 +32,10 @@ F : GeometricShapeFactory<P, A> {
         pedestrianGroup ?: Alone(this)
     }
 
+    init {
```

Comment:
I think that this class should simply get removed. Capabilities replace node types.